doc/source/general/cpu_hardware.rst

```diff
@@ -161,7 +177,3 @@ forums. You may put this code up near the top of your script::
       PRESERVE.
     }
 
-An Even Better Solution
-~~~~~~~~~~~~~~~~~~~~~~~
-
-There has been talk of instituting a special command: WAIT UNTIL PHYSICS that will sleep until there has been a physics update, and it's a good idea but it hasn't been implemented yet.
```

Comment:
I decided that this doesn't belong in the documentation anymore.  I put it there thinking this change was going to happen "any day now" and it's been like that for months so clearly it's not happening real soon.